src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Serialization/JsonFileDeserializer.cs

```diff
@@ -4,7 +4,7 @@
 using System.IO;
 using OmniSharp.Extensions.LanguageServer.Protocol.Serialization;
 
-namespace Microsoft.AspNetCore.Razor.LanguageServer
+namespace Microsoft.AspNetCore.Razor.LanguageServer.Serialization
```

Comment:
Moved for readability / organization. The code actions PR will contain other serializers in this namespace.